### PR TITLE
Bugfix: The name of ejected player not showind up in ejection screen

### PR DESCRIPTION
**Pull request by B28Rhein**
Fixed, just fixed, it was literally one symbol that broke it, no reason to write homes about it, it was probably added during privatization, ahh privatization, the worst, a lot of companies fell during privatization during balcerowicz's reforms, maybe they deserved it, I mean they probably wouldn't have bankrupted if they were making money before the privatization but dunno, maybe they fell because of bad management after privatization, companies that were bought by foreign investors didn't really look good for the government actually all of balcerowicz's period didn't look good for that government, there probably is a reason why they lost the next election... well dunno, weren't even born yet, wasn't supposed to write this much, as I said, easy fix not much to write about, bye.

### CODE REVIEW

**Review by SGGWAmongUs — APPROVED** [review]
TLDR, but thanks for contributing
